linux_sql/psql_docker.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+if [ $# != 2 ];
+then
```

Comment:
the second argument could be optional. the script will use the default 'password' as PGPASSWORD.